src/Nextjs.ts

```diff
@@ -114,14 +133,26 @@ export class Nextjs extends Construct {
         autoDeleteObjects: true,
       });
 
+    // layer
+    const nextLayer = new NextjsLayer(scope, 'NextjsLayer', {});
+
     // build nextjs app
     this.nextBuild = new NextjsBuild(this, id, { ...props, tempBuildDir });
     this.serverFunction = new NextJsLambda(this, 'Fn', {
       ...props,
+      nextLayer,
       tempBuildDir,
       nextBuild: this.nextBuild,
       lambda: props.defaults?.lambda,
     });
+    // build image optimization
+    this.imageOptimizationFunction = new ImageOptimizationLambda(this, 'ImgOptFn', {
+      ...props,
+      nextLayer,
+      imageConfig: props.imageOptimizationOptions?.imageConfig,
```

Comment:
add  tempBuildDir

Reply:
I added but removed since I didn't see it being used. I'll add it back in case.

Reply:
Actually I don't think it's needed now since I'm doing `extends NodejsFunction` now.